tools/ci_build/build.py

```diff
@@ -108,6 +108,7 @@ def parse_arguments():
     parser.add_argument("--use_tvm", action="store_true", help="Build with tvm")
     parser.add_argument("--use_openmp", action='store_true', help="Build with OpenMP.")
     parser.add_argument("--use_llvm", action="store_true", help="Build tvm with llvm")
+    parser.add_argument("--use_windowsthreadpool", action="store_true", help="Build with windowsthreadpool and taskthreadpool for windows")
```

Comment:
I didn't touch the code in MLAS yet. I believe that uses windows threadpool still … though I only added this in case we wanted to switch back at any point.. if you feel its unnecessary I can remove it.